src/styles/main.scss

```diff
@@ -12,16 +23,20 @@ html {
 body {
     display: flex;
     flex-direction: column;
-    font-size: 1.4rem;
+    font-size: 1.8rem;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-font-smoothing: antialiased;
```

Comment:
I don't see the issue in that screenshot.  

Yes, the font appears "thicker" or "brighter" between Photoshop and the browser.  It will appear completely different on Windows and Linux.

Be aware that different Operating Systems will render some fonts differently, especially with regard to the algorithms they use for antialiasing and character positioning.  Even different font formats will lead to differences in rendering algorithms, and user settings can also change it:
![image](https://cloud.githubusercontent.com/assets/1371232/13082878/7a32abc8-d4a0-11e5-956a-3d1655d96905.png)
![image](https://cloud.githubusercontent.com/assets/1371232/13082921/b9767f26-d4a0-11e5-9abc-5f9326af16e0.png)

(see: https://www.smashingmagazine.com/2012/04/a-closer-look-at-font-rendering/)

That's not something we can (or should try to) control.